### PR TITLE
Feature/testimonial

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: theme/css/style.css [hunk 1]
```diff
@@ -241,6 +242,28 @@ body {
...
+/* Testimonials */
+.testimonial-container .swiper-slide {
+  font-style: italic;
+  font-size: 15px;
+  flex-direction: column;
+  padding-top: 0;
+  margin-top: 0;
+}
+
+.testimonial-container .swiper-slide .who {
```
Comment (ElXyfer) [inline theme/css/style.css]: .swiper-slide class here not necessary

File: theme/css/swiper.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*testimonial */
+.testimonial-container.swiper-container {
+  width: 100%;
+  height: 450px;
+  padding-top: 10px;
+}
+
+.testimonial-container.swiper-container.full-width {
```
Comment (ElXyfer) [inline theme/css/swiper.css]: This part of the styling should be in the style.css file imo.

File: theme/css/swiper.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  width: 100vw;
+  margin-left: -50vw;
+  margin-right: -50vw;
+}
+
+.testimonial-container.swiper-container .swiper-wrapper {
```
Comment (ElXyfer) [inline theme/css/swiper.css]: You can get rid of this, its not doing anything.

File: theme/css/swiper.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  -ms-flex-align: center;
+  -webkit-align-items: center;
+  align-items: center;
+  border-radius: 10px;
+  transition: all .4s ease-in-out;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  padding: 5px 20px;
+}
+
+.swiper-slide:hover {
```
Comment (ElXyfer) [inline theme/css/swiper.css]: This needs to be more specific, include the testimonial container class.

File: index.html [hunk 5]
```diff
@@ -5,6 +5,10 @@
 	<link rel="stylesheet" href="fonts/Quicksand">
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 	<link rel="stylesheet" href="theme/css/style.css">
+	<script src="theme/node_modules/swiper/dist/js/swiper.min.js"></script>
+	<link rel="swiper" href="theme/css/swiper.css">
```
[the remark this answers is not in the document]
Comment (katlazarova) [reply index.html]: done

File: theme/css/swiper.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+/*testimonial */
+.swiper-container {
```
[the remark this answers is not in the document]
Comment (katlazarova) [reply theme/css/swiper.css]: done

File: ..gitignore [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
```
[the remark this answers is not in the document]
Comment (katlazarova) [reply ..gitignore]: deleted

File: .gitignore [hunk 8]
```diff
@@ -13,4 +13,5 @@
 .idea
 .idea/*
 learning.html
-ajax
\ No newline at end of file
+ajax
+theme/node_modules
```
[the remark this answers is not in the document]
Comment (katlazarova) [reply .gitignore]: done

File: theme/css/style.css (hunk 1 above)
Comment (katlazarova) [reply theme/css/style.css]: okay done

File: theme/css/swiper.css (hunk 2 above)
Comment (katlazarova) [reply theme/css/swiper.css]: okay

File: theme/css/swiper.css (hunk 3 above)
Comment (katlazarova) [reply theme/css/swiper.css]: done

File: theme/css/swiper.css (hunk 4 above)
Comment (katlazarova) [reply theme/css/swiper.css]: done

**Pull request merged**